ci/ray_ci/windows/build_base.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -ex
+
+conda init 
+pip install -U --ignore-installed  \
+  -c python/requirements_compiled.txt \
+  -r python/requirements.txt \
+  -r python/requirements/test-requirements.txt
+pip install torch==2.0.1 torchvision==0.15.2 \
+  tensorflow==2.11.0 tensorflow-probability==0.19.0
```

Comment:
what are these? why are these not using constraints? if you know, could you add a comment?

Reply:
will do, this clones how ci/env/install-dependencies.sh works https://github.com/ray-project/ray/blob/master/ci/env/install-dependencies.sh#L443; but I think i do things in wrong order